src/test/Number.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {shallow} from 'enzyme'
+
+import Number from '../components/widgets/Number'
+
+describe('Number widget component', () => {
+  const mainStat = '125'
+  const goal = '250'
+  const progress = ((mainStat / goal) * 100).toString().concat('%')
+
+  const widgetData = {
+        type: 'Number',
+        title: 'Number',
+        size: '1x1',
+        x: '10px',
+        y: '500px',
+        mainStat: mainStat,
+        goal: goal
+      }
+
+  const numberWrapper = shallow( <Number {...widgetData} /> )
+
+  it('renders without crashing', () => {
+    const numberSnapshot = renderer.create( <Number {...widgetData} /> ).toJSON()
+    expect(numberSnapshot).toMatchSnapshot()
+  })
+
+  it('renders the main stat', () => {
+    let mainStatDiv = numberWrapper.find('.main-stat')
+    expect(mainStatDiv.type()).toEqual('div')
+    expect(mainStatDiv.text()).toEqual(mainStat)
+  })
+
+  it('renders the goal', () => {
+    let goalDiv = numberWrapper.find('.goal-stat')
+    expect(goalDiv.type()).toEqual('div')
+    expect(goalDiv.text()).toEqual(goal)
+  })
+
+  it('renders the progress', () => {
+    let progressDiv = numberWrapper.find('.progress-stat')
+    expect(progressDiv.type()).toEqual('div')
+    expect(progressDiv.text()).toEqual(progress)
+  })
+
+  it('calcutates the progress', () => {
+    expect(
+      numberWrapper.instance().calculateProgress(
```

Comment:
If this is less than 80 chars, please collapse to one line.  This is challenging to read.